ast/src/statements/console/console_function.rs

```diff
@@ -20,10 +20,17 @@ use leo_span::Span;
 use serde::{Deserialize, Serialize};
 use std::fmt;
 
+/// A console logging function to invoke.
```

Comment:
The ABNF grammar also has `debug`. Is that not a thing any more? (Because in that case we should update the grammar.)